internal/aws/containerinsight/const.go

```diff
@@ -12,6 +12,11 @@ const (
 	// We assume 50 micro-seconds is the minimal gap between two collected data sample to be valid to calculate delta
 	MinTimeDiff = 50 * time.Microsecond
 
+	// Environment variables
+	RunInContainer            = "RUN_IN_CONTAINER"
+	RunAsHostProcessContainer = "RUN_AS_HOST_PROCESS_CONTAINER"
+	TrueValue                 = "True"
```

Comment:
I just noticed this one, probably the same in pull request 665
is it necessary to create a constant out of `True`? Is it necessary for this to be exported? (Capital T?)

Reply:
Okay, I see that this is already defined in the agent repo.  So you are maintaining the pattern, that is a plus.  My two cents is that we don't need the code to be the exact same and exporting this new variable just for this file is kind of weird

Reply:
Yes, it was necessary to be with capital `T` since CW-agent repo follows same pattern.
All these variables will be removed once we upgrade containerd to `1.7`